action.yml

```diff
@@ -46,6 +46,8 @@ runs:
             }
           }
 
+          core.setFailed('The request was not successful')
```

Comment:
I think it should be exit with 0 in successful case if you want to fail like this.

Reply:
Oh..., I missed it. I will fix it immediately.